Use Convex for trelleux example database

thomasballinger opened the pull request.

thomasballinger committed:
Add a Convex database trellaux example

autofix-troubleshooter commented:
Hi! I'm the <a href="https://autofix.ci"><img alt="autofix logo" src="https://autofix.ci/logo/logo.svg" width="16"><strong>autofix.ci</strong></a> troubleshooter bot.

It looks like you correctly set up a CI job that uses the autofix.ci GitHub Action, but the [**autofix.ci GitHub App**](https://github.com/apps/autofix-ci/) has not been installed for this repository. This means that autofix.ci unfortunately does not have the permissions to fix this pull request. If you are the repository owner, please install the app and then restart the CI workflow! 😃

thomasballinger committed:
examples(start-trellaux): fix re-ordering columns (#2061)

loyalesoul committed:
docs: fix a grammar error in the "code-splitting" guide (#2064)

chorobin committed:
fix(react-router): when navigating from a index it route it should resolve to a valid route (#2066)

tannerlinsley committed:
release: v1.45.16

rijk committed:
docs: missing comma in the Start Vinxi scripts (#2067)

* docs: fixed missing comma in Start Vinxi scripts

Also opted to change the `// ...` into a normal comment to avoid confusion.

* ci: apply automated fixes

---------

Co-authored-by: autofix-ci[bot] <114827586+autofix-ci[bot]@users.noreply.github.com>

SeanCassiere committed:
feat(react-router): support for `preload='viewport'` (#2062)

* feat(react-router): expand the types to allow the preload setting of "viewport"

* feat(react-router): add the `useIntersectionObserver` and `useForwardedRef` hooks

* feat(react-router): add the configuration to use `preload='viewport'`

* chore(react-router): cleanup `console.log`

* chore(react-router): types I can be happy with

* refactor(react-router): make the `useIntersectionObserver` hook testing compatible

* test(react-router): testing for preload='viewport'

* test(react-router): move the `IntersectionObserver` mock into the utils

* docs: update the preloading guide

* docs: api reference

* chore: apply suggestions from code review

tannerlinsley committed:
release: v1.46.0

SeanCassiere committed:
chore: cleanup the old `with-trpc` and `with-framer-motion` examples (#2068)

* chore: fixup the example

* chore: rename from `wip-with-framer-motion` to `with-framer-motion`

* chore: remove the `wip-with-bling` example

* chore: remove the `wip-waku-rsc` example

* chore: fixup the `with-trpc` example

* chore: remove the `kitchen-sink-react-query` example

* Revert "chore: remove the `kitchen-sink-react-query` example"

This reverts commit 2fd8af97ad448dcbf944a84432f025a7a12a87d1.

autofix-ci[bot] committed:
ci: apply automated fixes
